### PR TITLE
Ghost Pinbot peer ID exclusion

### PR DESCRIPTION
**Pull request by simonovic86**
- exclude Ghost Pinbot peer ID from members list
- implement unit test
- fix hanging issue with existing Ghost Thread tests

### CODE REVIEW

File: src/3box.js [hunk 1]
```diff
@@ -245,6 +248,12 @@ class Box extends BoxApi {
     if (!this.spaces[space]) {
       this.spaces[space] = new Space(space, this.replicator)
     }
+
+    if (this._ghostPinbot) {
+      Object.assign(opts, {
+        ghostPinbot: this._ghostPinbot
+      })
```
Comment (oed) [inline src/3box.js:255]: I'd just do `opts.ghostPinbot = this._ghostPinbot` easier to read :)

File: src/space.js [hunk 2]
```diff
@@ -167,6 +167,7 @@ class Space {
    * @param     {Boolean}   opts.confidential       create a confidential thread with true or join existing confidential thread with an encKeyId string
    * @param     {Boolean}   opts.noAutoSub          Disable auto subscription to the thread when posting to it (default false)
    * @param     {Boolean}   opts.ghost              Enable ephemeral messaging via Ghost Thread
+   * @param     {String}    opts.ghostPinbot        Enable ephemeral messaging via Ghost Thread
```
Comment (oed) [inline src/space.js:170]: Should say something like: `Multiaddress of a ghostpinbot instance`

File: src/__tests__/ghost.test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 jest.mock('3id-resolver', () => {
   const { didResolverMock } = require('../__mocks__/3ID')
   return {
-    getResolver: () => ({'3': didResolverMock})
+    getResolver: () => ({ 3: didResolverMock })
```
[the remark this answers is not in the document]
Comment (simonovic86) [reply src/__tests__/ghost.test.js:4]: It's an unnecessarily quoted property but I can change it back.

File: src/3box.js (hunk 1 above)
Comment (simonovic86) [reply src/3box.js]: done 👍

File: src/space.js (hunk 2 above)
Comment (simonovic86) [reply src/space.js]: oh, copied from above, thanks! fixed 👍

File: src/__tests__/ghost.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (simonovic86) [reply src/__tests__/ghost.test.js]: changed 👍

**Pull request merged**